gitpod-io/gitpod
# publish gitpod-protocol and supervisor-api to npm

This PR publishes gitpod-protocol and supervisor api to NPM to consume be 3rd party IDEs:
- each build publishes new artefacts using a namespace as a tag
- latest tag should be set during the official release to a concrete version (there are no yet release scripts for it)
- master tag can be used to consume latest unstable changes

##### How to test

Create package.json and add  `@gitpod/gitpod-protocol@ak-npm-publish` as a dependency. Check content of node_modules afterwards.

<img width="1038" alt="Screenshot 2020-09-22 at 07 24 13" src="https://user-images.githubusercontent.com/3082655/93846551-a74e8b00-fca4-11ea-8e50-b836329e3f7d.png">
